### ISSUE TITLE
Windows: Stuck on 'starting' when unable to reach control server

### ISSUE THREAD

**Issue by Haarolean** [tailscale/tailscale#13660]
### What is the issue?

I'm running tailscale on windows in a network where tailscale is actively restricted. I had one account set up and wanted to add a different (headscale) one, but the client got stuck, clicking on 'add account' or reauthenticate did nothing. Had to restart, and upon start the client was stuck in 'starting...'. 

Some logs: https://gist.github.com/Haarolean/6aa3f1943b30d37ed7c30a0adeb009a8
Let me know if you need full logs, not sure if they contain any sensitive information and, thus not included.

### OS

Windows

### OS version

win 11

### Tailscale version

1.74.0

### Bug report

BUG-fdd27a9edad373ec02dea5d79e4c30e8ebc585cead58186f7784d0ffbae06d43-20241002222100Z-4c8c5c9ac337a3d3

**Comment by Haarolean** [comment]
Nuking config files to get rid of an existing profile doesn't help, a clean restart greets me with "needs authentication", clicking on "log in" does nothing, there's no account settings pane or whatsoever like on macOS, so the tailscale becomes completely unusable.

**Comment by rgblack316** [comment]
I am seeing the same thing after the Windows 11 24H2 update at the end of last week.

**Comment by dblohm7** [comment]
Yeah, that state could use some improvement. We've got some work coming up later this year to make that status more informative and precise.

**Comment by Haarolean** [comment]
> Yeah, that state could use some improvement. We've got some work coming up later this year to make that status more informative and precise.

It's not just about observability. The app is not functioning at all in case of a 451.

**Comment by isotisot** [comment]
Also having issues with ver 1.76.0 stuck on starting after upgrading to Win 11 24H2.

**Comment by ls2sl** [comment]
ver 1.76.0 on win10 also have this issue

**Comment by EstamelGG** [comment]
This will help reauth the client, but yes, after we restart tailscale or reboot Windows, it will stuck again.

```powershell
Stop-Process -Name "tailscale-ipn" -Force -ErrorAction SilentlyContinue

Stop-Service -Name "tailscale" -Force

$service = Get-Service -Name "tailscale"
while ($service.Status -ne 'Stopped') {
    Start-Sleep -Seconds 3
    $service.Refresh()
}

Stop-Process -Name "tailscaled" -Force -ErrorAction SilentlyContinue

Remove-Item -Path "C:\ProgramData\Tailscale" -Recurse -Force
Remove-Item -Path "$env:LOCALAPPDATA\Tailscale" -Recurse -Force

Start-Service -Name "tailscale"
```

**Comment by Ling-YangHui** [comment]
ver 1.76.3 on windows 11 24H2 also has this issue

**Comment by BSalita** [comment]
Stuck on start issue got resolved when I uninstalled Cloudflare Warp. W11 24H2, TS 1.76.6, Warp 2024.9.346.0. I never noticed any conflict issue before but I can't say for sure if this is new behavior. The hang only seems to occur if Tailscale is started after Warp is engaged.

**Comment by skakdollar** [comment]
Have the same issue with startup/reconnect with a custom control plane.

PS C:\Users\me> tailscale status
# Health check:
#     - Tailscale failed to set the DNS configuration of your device: rename C:\WINDOWS\system32\drivers\etc\hosts.tmp634006346 C:\WINDOWS\system32\drivers\etc\hosts: Access is denied.
#     - rename C:\WINDOWS\system32\drivers\etc\hosts.tmp634006346 C:\WINDOWS\system32\drivers\etc\hosts: Access is denied.

unexpected state: NoState

**Comment by hallelujah-shih** [comment]
also on ver 1.78.1

**Comment by iimamul** [comment]
> Stuck on start issue got resolved when I uninstalled Cloudflare Warp. W11 24H2, TS 1.76.6, Warp 2024.9.346.0. I never noticed any conflict issue before but I can't say for sure if this is new behavior. The hang only seems to occur if Tailscale is started after Warp is engaged.

I could also fix this issue by uninstalling Cloudflare Warp.

**Comment by hallelujah-shih** [comment]
> also on ver 1.78.1

on myvwin 11, into task-manager, restart tailscale service, successed.

![image](https://github.com/user-attachments/assets/53e50f0b-ac06-4704-bdaf-36d5d3d0ecb7)

**Comment by clemente3905** [comment]
Same issue for me. Win11, tailscale always stuck on "Starting".
No indication on what to do.
I lost remote access to any device on my tail net...

**Comment by wpattison74** [comment]
Search brought me here because I have the same issue - stuck on startup, Win11. Then I realized SurfShark (VPN) auto-updated and it's on by default. Once I disconnected that, Tailscale connected no issue.

**Comment by kites262** [comment]
Windows 11 24H2, same

**Comment by xiaohongred** [comment]
pve hosted  vm  win10 22H2 , same issue

**Comment by raggi** [comment]
If you arrive here because you were searching about some symptoms, but are not in a situation that matches the ticket description, please reach out to support: https://tailscale.com/contact/support#support-form or file a new bug that describes your specific symptoms and includes a bug report.
